.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     - name: Docker dependencies
       if: ${{ matrix.os == 'ubuntu-latest' }}
       run: |
-        docker-compose up -d
+        docker compose up -d
```

Comment:
If possible I would prefer to make changes separately, lets focus if you dont mind on jus the `stats` command

Reply:
Ok